ptbr/data.py

```
@@ -0,0 +1,256 @@
+                {
+                    key
+                    for item in raw
+                    if isinstance(item, dict)
+                    for key in item
+                }
+            )
+            available = ", ".join(repr(col) for col in available_columns) or "<none>"
+            raise ValueError(
+                f"Missing required column(s): {missing}. "
+                f"Available columns in local file: {available}"
+            )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)